src/rules/no-property-change-update.ts

```diff
@@ -51,11 +51,7 @@ const rule: Rule.RuleModule = {
         return;
       }
 
-      const props = getPropertyMap(node);
-
-      if (props) {
-        propertyMap = props;
-      }
+      propertyMap = getPropertyMap(node);
```

Comment:
what happened to the check here? so before we were only setting the property map when it was a non-null value, but now we always set it

Reply:
honestly not sure... 🤔